packages/moonstone/TooltipDecorator/TooltipDecorator.js

```diff
@@ -443,7 +377,8 @@ const TooltipDecorator = hoc(defaultConfig, (config, Wrapped) => {
 					onBlur: this.handleBlur,
 					onFocus: this.handleFocus,
 					onMouseOut: this.handleMouseOut,
-					onMouseOver: this.handleMouseOver
+					onMouseOver: this.handleMouseOver,
+					onKeyDown: this.handleKeyDown
```

Comment:
Why have you added a key down handler in props with a global handler already attached?

Reply:
I separated the keydown events for two different cases; I left notes/comments on L280 and L289 let me know if this works!